adafruit_servokit.py

```diff
@@ -61,8 +61,7 @@ class ServoKit:
 
     :param int channels: The number of servo channels available. Must be 8 or 16. The FeatherWing
                          has 8 channels. The Shield, HAT, and Bonnet have 16 channels.
-    :param ~I2C i2c: The I2C bus to use. If not provided, it will use generate the default I2C
-                                      bus singleton ``busio.I2C()`` and use that.
+    :param ~I2C i2c: The I2C bus to use. If not provided, the default I2C ``board.I2C()`` will be used.
```

Comment:
Looks like this just needs to be split into two lines like some of the other docstring params here.